unit-tests/live/options/test-drops-on-set.py

```diff
@@ -108,7 +108,7 @@ def check_color_frame_drops(frame):
 # inter_cam_sync_mode --> frame drops are expected during inter_cam_sync_mode change
 # emitter_frequency   --> Not allowed to be set during streaming
 if product_line == "D400":
-    options_to_ignore = [rs.option.visual_preset, rs.option.inter_cam_sync_mode, rs.option.emitter_frequency]
+    options_to_ignore = [rs.option.visual_preset, rs.option.inter_cam_sync_mode, rs.option.emitter_frequency, rs.option.auto_exposure_mode]
```

Comment:
The lines above explain why each is ignored.
Why not auto_exposure_mode??

Reply:
Good catch, I will add